rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2288,6 +2288,10 @@ def test_upgrade_db_41_to_42(user_data_dir, messages_aggregator):
             SupportedBlockchain.deserialize(x) for x in json.loads(raw_list)
         ] == [SupportedBlockchain.POLYGON_POS, SupportedBlockchain.GNOSIS]
 
+        # get current oracles and check that manualcurrent was removed and all others remain.
+        settings = db.get_settings(cursor=cursor)
+        assert CurrentPriceOracle.MANUALCURRENT not in settings.current_price_oracles
```

Comment:
this test is not valid yet. That's because `db.get_settings` returns settings that are stored in Db + default values for settings that are not stored. So, if you check `rotkehlchen/db/settings.py` line 186, the default is used without check the DB table.

Reply:
Test should be like this:
1. Add the setting first in the table before upgrade. Can also add it directly in `rotkehlchen/tests/data/v41_rotkehlchen.db` (which is even better, because it saves us from doing an extra DB query). Also, if you edit any DB in the codebase, comment a SQL diff of it in the PR.
2. Ensure it's added by doing a SELECT query.
3. Upgrade
4. Check that the manualcurrent is removed.